src/pages/registerPage.ts

```diff
@@ -0,0 +1,87 @@
+import { Page, expect } from "@playwright/test";
+
+const LocatFirstName = "#input-firstname";
+const LocatLastName = "input[name='lastname']";
+const LocatEmail = "input[name='email']";
+const LocatTelephone = "input[name='telephone']";
+const LocatPassword = "input[name='password']";
+const LocatConfirPassword = "input[name='confirm']";
+const LocatCheck = "#input-newsletter-no";
+const LocatPrivacy = "//label[@for='input-agree']";
+const LocatContinue = "input[value='Continue']";
+
+const LocatDangerFirstName = "//input[@name='firstname']/following-sibling::div[1]";
+const LocatDangerLastName = "//input[@name='lastname']/following-sibling::div[1]";
+const LocatDangerEmail = "//input[@name='email']/following-sibling::div[1]";
+const LocatDangerTelephone = "//input[@name='telephone']/following-sibling::div[1]";
+const LocatDangerPassword = "//input[@name='password']/following-sibling::div[1]";

```

Comment:
not that critical, but why do you use mix of css and xpath locators? 
if you use css, please use css otherwise use xpath.

Reply:
I tried many different options in this case, but the values were unstable. I originally used the values .text-danger or (//div[@class='text-danger'])[1]  depending on the case. I think about how to fix it.